index.bs

```diff
@@ -1986,24 +2112,34 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |options| be the value of <code>|options|.{{CredentialRequestOptions/publicKey}}</code>.
 
-1. If the {{PublicKeyCredentialRequestOptions/timeout}} member of |options| is present, check if its value lies
-    within a reasonable range as defined by the [=client=] and if not, correct it to the closest value lying within that range.
-    Set a timer |lifetimeTimer| to this adjusted value. If the {{PublicKeyCredentialRequestOptions/timeout}} member of
-    |options| is not present, then set |lifetimeTimer| to a [=client=]-specific default.
+1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is present with the value
+    {{CredentialMediationRequirement/conditional}}:
 
-        Recommended ranges and defaults for the {{PublicKeyCredentialRequestOptions/timeout}} member of |options| are as follows.
-          If <code>|options|.{{PublicKeyCredentialRequestOptions/userVerification}}</code>
-            <dl class="switch">
-                :   is set to {{UserVerificationRequirement/discouraged}}
-                ::  Recommended range: 30000 milliseconds to 180000 milliseconds.
-                ::  Recommended default value: 120000 milliseconds (2 minutes).
+    1. If <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> is not [=list/empty=],
+        return a {{DOMException}} whose name is "{{NotAllowedError}}", and terminate this algorithm.
 
-                :   is set to {{UserVerificationRequirement/required}} or {{UserVerificationRequirement/preferred}}
-                ::  Recommended range: 30000 milliseconds to 600000 milliseconds.
-                ::  Recommended default value: 300000 milliseconds (5 minutes).
-            </dl>
+    1. Set a timer |lifetimeTimer| to a value of infinity.
```

Comment:
It's only partly because of that (if that was the only reason, we could keep the timer). The main reason is we don't want the credentials to suddenly stop appearing on the autofill prompt after some time has passed. That would be extremely confusing for the user. I'll add a note explaining this.